### ISSUE TITLE
QUIC is not supressed on AG for Android

### ISSUE THREAD

**Issue by sfionov** [AdguardTeam/CoreLibs#750]
```
14:34:40.502 [tcpip-stack-pool-1-thread-89] DEBUG c.a.a.filtering.api.LocalVpnService - UDP id=6 onUdpConnectRequest: protocol=UDP, local=/172.18.11.218:59344, remote=/173.194.44.19:443
14:34:40.505 [tcpip-stack-pool-1-thread-89] DEBUG c.a.android.filtering.events.i - Creating new stats for com.android.chrome with id 6 and network type MOBILE
```
Not passed as "QUIC" in connect request

**Comment by sfionov** [comment]
core/pull-requests/1071

**Comment by sfionov** [comment]
For QA:
Google should be filtered.
Filtering log should not show not blocked entries like udp://x.x.x.x:443 from Chrome

**Comment by TPS** [comment]
I think this is now generally implemented in the latest ηightly.  Could there also be a corresponding method to generally _allow_ QUIC (i.e., *dis*able this setting, perhaps via an advanced setting)?

**Comment by ameshkov** [comment]
@TPS but for what reason? Tbh, I suppose we should add QUIC and HTTP/3 filtering in the future, thus we won't have to suppress it.

**Comment by TPS** [comment]
Until #487 is implemented (#746 is equivalent to _this_ issue on Mac), a debug method to allow QUIC might help debug behavior, e.g., that I suspect has to do w/ this.
